### ISSUE TITLE
Update ELI5 - Interledger Protocol

### ISSUE THREAD

**Issue by melissahenderson** [interledger/interledger.org-developers#42]
Sabine has asked us to update and revive the following and make it accessible on the Interledger dev site.

https://github.com/interledger/rfcs/blob/main/deprecated/0020-explain-like-im-five/0020-explain-like-im-five.md

Jeremiah has linked to this ELI5 a few times re: Web Monetization. It may be a good one to review for ideas. https://developers.mobilecoin.com/learn/explain-like-i%27m-five/fog/

**Comment by huijing** [comment]
Can we do this with the lollipops and magic diagram somehow?

**Comment by hajjimo** [comment]
Indeed, using lollipops and magic diagrams will help explain this to simulated 5-year-olds.  We're putting a pause on ILP stuff until after the Rafiki docs have been polished and republished.
